decidim-pages/db/migrate/20161214150429_add_commentable_to_pages.rb

```diff
@@ -0,0 +1,5 @@
+class AddCommentableToPages < ActiveRecord::Migration[5.0]
+  def change
+    add_column :decidim_pages_pages, :commentable, :boolean, null: false, default: false
```

Comment:
@josepjaume, after talking offline with @beagleknight: why are we adding this to the page instead of the feature? If we want to do the same for proposals or meeting we'll have to duplicate this code. What do you think of moving it to a feature?